apps/site/pages/en/blog/events/collab-summit-2024-dublin.md

```diff
@@ -0,0 +1,227 @@
+- **Cross-Posting**: There was a suggestion to start with cross-posting to both platforms to ensure a smooth transition and maintain engagement with existing followers.
+- **Automation**: Jacob mentioned that Bluesky supports automation, which could be beneficial for managing social media presence.
+- **Foundation Involvement**: The discussion highlighted the need to involve the OpenJS Foundation in this decision and potentially take it to the Community Programs Committee (CPC) for further deliberation.
+- **Password Sharing**: Joyee suggested sharing the social media account passwords using a secure method like OnePassword to streamline the posting process and reduce delays due to timezone differences.
+
+### Handling Experimental Status
+
+- **Experimental Features**: The group discussed the handling of experimental features, especially when their adoption becomes significant. Stephen noted that even though some features are experimental, they are widely used by the community, such as `module.register`.
+- **Timeline and Expectations**: James emphasized the importance of setting clear expectations and timelines for experimental features. This includes providing warnings before major changes and ensuring clear documentation about the status and upcoming changes.
+- **Opt-In Mechanisms**: There was a discussion on making experimental features opt-in via API or flags, especially for library authors who might rely on these features.
+- **Lint Rules and Codemods**: Wes suggested using lint rules to enforce best practices and codemods to facilitate migrations. Jordan cautioned that changing people's behavior is challenging and requires incentives rather than enforcement.
+
+### Facilitating Migrations
+
+- **Codemods**: Jacob demonstrated the `ts-correct-specifier` codemod, which can help automate the migration process. The group discussed the potential for codemods to be used not just for code but also for dependencies.
+- **Dependabot Integration**: Darcy suggested improved Dependabot integration to facilitate migrations, noting that people may not want to run random codemods found on the internet.
+- **Registry for Migrated Projects**: The idea of maintaining a registry for projects that have already been migrated was proposed to avoid redundant work and save compute resources.
+- **Good First Issues**: Geoffrey suggested creating a tracking issue in the Node.js repository for codemods that need to be developed, tagging them as "good first issues" to encourage new contributors.
+
+## Personal note
+
+> I think that was the part that touched and pleased me the most. It was the connection with the people. Because everyone involved is great. These are people I'd only been able to talk to online, and here you meet them in real life.
+>
+> <cite>Augustin Mauroy</cite>
```

Comment:
I don't want to change your quote to much, because it's not mine to change, but a slight adjustment on wording could be:

```suggestion
> I think the part that touched and pleased me the most was the connection with the people, because everyone involved is great. These are people I'd only been able to talk to online, and here you meet them in real life.
>
> <cite>Augustin Mauroy</cite>
```

Reply:
Thanks for helping on writing